Automattic/jetpack
# Google Calendar Block: Fix styling issues with new site editor

Related to: 256-gh-Automattic/view-design

This change updates the CSS for the editor view of the Google Calendar block to create consistency between the post editor and the new FSE site editor.

Note that it doesn't address the ExternalLink icon sizing issue, which is raised in: https://github.com/WordPress/gutenberg/issues/30553

#### Changes proposed in this Pull Request:

* In the Google Calendar block, update the textarea's font family and font size to be set to `inherit` to match the behaviour in the post editor (which expects `form.css` to be present, which it isn't in the site editor)
* Ensure the form bottom margin is consistent with the post editor
* Remove the border from the iframe when inserted in the site editor

#### Screenshots

#### Editor placeholder

Note the default font for the textarea, and the bottom margin on the form has been removed

| Before | After |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/14988353/113824392-12842880-97c3-11eb-8ae9-d6e7f7ee640b.png) | ![image](https://user-images.githubusercontent.com/14988353/113824266-ea94c500-97c2-11eb-814e-413b0f77c772.png) |

#### With a calendar embed

Note the border has been removed.

| Before | After |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/14988353/113824179-cdf88d00-97c2-11eb-96a5-acb5037a4a01.png) | ![image](https://user-images.githubusercontent.com/14988353/113824204-d4870480-97c2-11eb-91d7-e2b32d6bbd67.png) |

#### Jetpack product discussion


Related to: 256-gh-Automattic/view-design

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* With the Gutenberg plugin installed and a block-based theme activated (e.g. tt1-blocks), navigate to the site editor (`/wp-admin/admin.php?page=gutenberg-edit-site`), and:
* Try inserting a Google Calendar block in the Footer template part
* Compare against the placeholder state in the post/page editor (the only major difference should be the icon size for the external links (raised in https://github.com/WordPress/gutenberg/issues/30553)
* Embed a public Google Calendar url (e.g. here's my test calendar url that you can use: https://calendar.google.com/calendar/embed?src=c_9rjat37j3hl929dalnbovb00ps%40group.calendar.google.com&ctz=Australia%2FSydney )
